packages/devtools_app/lib/src/connected_app.dart

```diff
@@ -55,10 +85,16 @@ class ConnectedApp {
   Future<bool> _libraryUriAvailable(String uri) async {
     assert(serviceManager.serviceAvailable.isCompleted);
     await serviceManager.isolateManager.selectedIsolateAvailable.future;
-
     return serviceManager.isolateManager.selectedIsolateLibraries
         .map((ref) => ref.uri)
         .toList()
         .any((u) => u.startsWith(uri));
   }
+
+  Future<void> initializeValues() async {
+    await isFlutterApp;
+    await isProfileBuild;
+    await isDartWebApp;
+    _appTypeKnown = true;
```

Comment:
why is this set to true here as well as using the _updateAppTypeKnown getter in other cases?
Consider executing these three async requests in parallel for slightly faster startup.
E.g.
`await Futures.wait([isFlutterApp, isProfileBuild, isDartWebApp]);`
so that all vm requests are triggered simultaneously. would only really matter if remote debugging.

Reply:
done